extras/ds3231/ds3231.c

```diff
@@ -0,0 +1,346 @@
+#include "ds3231.h"
```

Comment:
Could you add a copyright/license header here & on ds3231.h please? Doesn't have to be the full license, just a short one like for the example would be fine.